### PR TITLE
AC-8362 Reflect more inclusive options for gender

### PR DESCRIPTION
**Pull request by geofrocker**
### Changes Introduced in [AC-8362 ](https://masschallenge.atlassian.net/browse/AC-8362 ):
- A schema migration for the gender object
- A data migration to add the options provided by the business
- Updated factories
### Testing:
- Check out this branch
- On impact, run `docker-compose run --rm web ./manage.py migrate accelerator`
- Navigate to one of the profiles on the admin page e.g(this [profile](http://localhost:8181/admin/mc/expertprofile/79/change/))
- Notice the gender fields
- Confirm that the changes are consistent with the ticket description

### CODE REVIEW

**Comment by freshprincekla** [comment]
There are some linting errors on this PR.

File: accelerator_abstract/models/base_gender_choices.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# MIT License
+# Copyright (c) 2017 MassChallenge, Inc.
+
+from __future__ import unicode_literals
+
+from django.db import models
+
+from accelerator_abstract.models.accelerator_model import AcceleratorModel
+
+GENDER_MALE_CHOICE = "Male"
```
Comment (jonkiparsky) [inline accelerator_abstract/models/base_gender_choices.py:10]: Why are these specified here? Gender choices should be specified by staff in the database.

File: settings.py [hunk 2]
```diff
@@ -107,6 +107,8 @@
 ACCELERATOR_PAYPALPAYMENT_MODEL = "accelerator.PayPalPayment"
 ACCELERATOR_LEGALCHECK_MODEL = "accelerator.LegalCheck"
 ACCELERATOR_USERLEGALCHECK_MODEL = "accelerator.UserLegalCheck"
+ACCELERATOR_USERLEGALCHECK_MODEL = "accelerator.UserLegalCheck"
```
Comment (jonkiparsky) [inline settings.py:110]: duplicated line

**Review by jonkiparsky — APPROVED** [review]

**Review by iankigen — APPROVED** [review]
QA/Code looks good

**Pull request merged**